src/main/java/com/avairebot/commands/music/RepeatMusicQueueCommand.java

```diff
@@ -79,6 +79,9 @@ public String getDescription() {
 
     @Override
     public boolean onCommand(CommandMessage context, String[] args) {
+        if (args.length == 0) {
+            return sendErrorMessage(context, context.i18n("invalidArguments"));
```

Comment:
You can replace this line with
```java
return sendErrorMessage(context, "errors.missingArgument", "option");
```
Where `option` is the thing people are missing, it will use the global error messages from the language files to produce the string
```
Missing argument, the `{0}` argument is required!
```
Where `{0}` would be replaced with `option`, it's just one less line that needs to be translated in the end.